src/schematools/permissions/__init__.py

```diff
@@ -1,3 +1,5 @@
+"""Base package for permissions."""
```

Comment:
Side remark, but this doc isn't very informative. If this is to satisfy a checker, maybe we should get set it to a laxer mode.

Reply:
Agree.

Reply:
I laxed this flake8 check in setup.cfg.